docs/ops/doc/ParameterConversion.rst

```diff
@@ -109,10 +109,9 @@ Using this ``engine.convert`` Op, SciJava Ops can match our ``filter.convolve``
 
 At runtime, the Op matcher will invoke the following steps:
 
-* The ``Img<DoubleType> input`` is left alone, as it is already of the type expected by the Op.
+* The ``Img<DoubleType> in`` is left alone, as it is already of the type expected by the Op.
 * The ``double[][] kernel`` is converted to a ``RandomAccessibleInterval<DoubleType> kernel1`` using our ``engine.convert`` Op.
-* The Op convolves ``input1`` with ``kernel1``, returning an ``Img<DoubleType> output1``
-* The ``Img<DoubleType> input1`` is left alone and returned to the user, as it is already of the type expected by the user.
+* The Op convolves ``input`` with ``kernel1``, returning an ``Img<DoubleType> result``.
```

Comment:
Maybe let's just make the user argument `in` and the Op parameter `input` the same?

Reply:
I pushed another commit trying to make the names as consistent as possible. I also pushed improvements to the mesh_viz example, and renamed the title of this PR to reflect the broadened scope.